### PR TITLE
re:knifes

### PR DESCRIPTION
**Pull request by rye-rice**
## About The Pull Request
Repsprites most knifes in the games, at least the ones that matter. Done on behalf of FleetingExpertise.
## Why It's Good For The Game

image

## Changelog

:cl: FleetingExpertise
add: Most knifes in the game have  been resprited!
/:cl:

### CODE REVIEW

File: code/game/objects/items/kitchen.dm [hunk 1]
```diff
@@ -66,10 +66,13 @@
 
 /obj/item/kitchen/knife
 	name = "kitchen knife"
-	icon_state = "knife"
-	item_state = "knife"
+	icon = 'icons/obj/item/knife.dmi'
+	lefthand_file = 'icons/mob/inhands/weapons/knifes_lefthand.dmi'
```
Comment (thgvr) [inline code/game/objects/items/kitchen.dm:70]:
```suggestion
	lefthand_file = 'icons/mob/inhands/weapons/knives_lefthand.dmi'
```
oops?

**Comment by thgvr** [comment]
w/e

**Comment by rye-rice** [comment]
Remember, no clue why checks failed

**Comment by thgvr** [comment]
same reason CI fails on prs

**Comment by FalloutFalcon** [comment]
" The plural form of ‘knife’ is ‘knives.’ "

**Pull request merged**